Plots/XY/NCL_tm_2.py

```diff
@@ -0,0 +1,113 @@
+"""
+NCL_tm_2.py
+===============
+Tickmark and Axis Manipulation example
+
+This Python script reproduces the NCL plot script found here: https://www.ncl.ucar.edu/Applications/Scripts/tm_2.ncl
+"""
+
+
+###############################################################################
+# Import the necessary python libraries
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.ticker import (MultipleLocator, FormatStrFormatter)
+
+###############################################################################
+# Define a function for creating an array of random values.
+
+
+def random_floats(low, high, size):
+    values = [np.random.uniform(low, high) for _ in range(size)]
+    return values
+
+
+###############################################################################
+# Create the plot data.
+# Note that range() top value is not included in the returned array of values.
+
+x_data = range(1950, 2006)
+y_data = random_floats(-4, 4, 56)
```

Comment:
And as mentioned, you can change this just to:

```suggestion
y_data = np.random.uniform(-4, 4, 56)
```